2022-07-17 18:44:29 [info]: API listening at http://localhost:9000
2022-07-17 19:11:05 [info]: ::1 (GET) /bots
2022-07-17 19:11:17 [info]: ::1 (GET) /bots
2022-07-17 19:11:20 [info]: ::1 (GET) /bots
2022-07-17 19:16:03 [info]: API listening at http://localhost:9000
2022-07-17 19:16:06 [info]: ::1 (GET) /bots
2022-07-17 19:16:12 [info]: ::1 (GET) /auth
2022-07-17 19:16:22 [info]: ::1 (GET) /bots
2022-07-17 19:18:29 [info]: ::1 (POST) /bots
2022-07-17 19:18:37 [info]: ::1 (POST) /bots
2022-07-17 19:18:41 [info]: ::1 (GET) /bots
2022-07-17 19:22:24 [info]: API listening at http://localhost:9000
2022-07-17 19:22:27 [info]: ::1 (GET) /bots
2022-07-17 19:32:54 [info]: ::1 (GET) /bots/e57a
2022-07-17 19:33:41 [info]: ::1 (PUT) /bots/e57a
2022-07-17 19:33:45 [info]: ::1 (GET) /bots/e57a
2022-07-17 19:34:30 [info]: ::1 (GET) /bots/e57a/tasks
2022-07-17 19:34:30 [error]: ReferenceError: links is not defined
    at C:\Users\larth\Coding\streamfinder\api\routes\bot.js:68:34
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2022-07-17 19:34:59 [info]: API listening at http://localhost:9000
2022-07-17 19:35:02 [info]: ::1 (GET) /bots/e57a/tasks
2022-07-17 19:48:24 [info]: ::1 (GET) /links
2022-07-17 19:49:00 [info]: ::1 (POST) /links
2022-07-17 19:49:07 [info]: ::1 (POST) /links
2022-07-17 19:49:07 [error]: TypeError: linkController.getTask is not a function
    at getNewID (C:\Users\larth\Coding\streamfinder\api\models\botlinks.js:56:27)
    at BotLinks.insert (C:\Users\larth\Coding\streamfinder\api\models\botlinks.js:34:26)
    at C:\Users\larth\Coding\streamfinder\api\model_functions\botlinks.js:36:22
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2022-07-17 19:49:31 [info]: API listening at http://localhost:9000
2022-07-17 19:49:33 [info]: ::1 (POST) /links
2022-07-17 19:49:40 [info]: ::1 (GET) /links
2022-07-17 19:50:54 [info]: ::1 (POST) /bots/e57a/tasks
2022-07-17 19:50:54 [error]: Error: ER_BAD_FIELD_ERROR: Unknown column 'finished' in 'field list'
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertTask (C:\Users\larth\Coding\streamfinder\api\controllers\dc.task.js:15:21)
    at Task.insert (C:\Users\larth\Coding\streamfinder\api\models\task.js:68:30)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2022-07-17 19:52:36 [info]: API listening at http://localhost:9000
2022-07-17 19:52:38 [info]: ::1 (POST) /bots/e57a/tasks
2022-07-17 19:52:42 [info]: ::1 (GET) /bots/e57a/tasks
